src/WickedReports/WickedReports.php

```diff
@@ -137,6 +138,25 @@ public function addOrderItems($items)
         return $this->request('orderitems', 'POST', $items);
     }
 
+    /**
+     * @param array|MarketingDataCollection $items
+     * @return bool|string
+     * @throws ValidationException
+     * @throws WickedReportsException
+     */
+    public function addMarketingData($items)
+    {
+        if (is_array($items)) {
+            $items = new MarketingDataCollection($items);
+        }
+
+        if ( ! $items instanceof MarketingDataCollection) {
+            throw new ValidationException('$items must be either array or MarketingDataCollection type');
```

Comment:
please change message to
```
'Items must be either array or MarketingDataCollection type'
```
Final user shouldn't care about our internal variables - only logic things.